autode/conformers/conformers.py

```diff
@@ -234,6 +237,65 @@ def prune_diff_graph(
         logger.info(f"Pruned on connectivity {n_prev_confs} -> {len(self)}")
         return None
 
+    def prune_with_boltzmann(
+        self, temperature: float = 298.15, threshold: float = 0.9
+    ) -> None:
+        """
+        Keep the lowest energy conformers whose cumulative weights are up to
+        the threshold (default 90%). Conformer energies must be in Ha.
+        -----------------------------------------------------------------------
+
+        Arguments:
+            threshold (float): Cumlative weight threshold.
+            temperature (float): Temperature in K.
+        """
+        assert threshold <= 1.0, "The treshold must be below 1.0"
+
+        hartrees_to_joules = 4.35974417 * 10**-18
```

Comment:
oo there's no need to do any manual conversions:

```python
>>> import autode as ade
>>> mol = ade.Molecule(smiles="C")
>>> mol.single_point(method=ade.methods.XTB())
>>> float(mol.energy)
-4.17244719832
>>> float(mol.energy.to("J"))
-1.8190806856428136e-17
```

Reply:
so can get rid of the assert on the units

Reply:
Very useful - thanks. Thought it might be possible but couldn't quite work out how to do it